client/engine/engine/ui/animation/state.py

```diff
@@ -1,6 +1,8 @@
 from ..state import UIElementState
 
 
+# TODO THIS SHOULD NOT BE STATE, THIS SHOULD BE MANAGED BY THE
+# LOGIC
 class AnimationState(UIElementState):
```

Comment:
This wasn't properly designed, in order to join static images and animations this should go in the logic